api/data_explorer/__main__.py

```diff
@@ -238,26 +217,38 @@ def _get_interval(max_field_value):
         return 1000000000000
 
 
-def _get_es_facets():
-    """Returns a dict from UI facet name to Elasticsearch facet object."""
+def _process_facets():
+    """Process facets to store a dict from UI facet name to UI facet description
```

Comment:
to UI facet description, Elasticsearch field name, and field type

Reply:
Done